src/lib.rs

```diff
@@ -39,42 +44,38 @@ pub use data_structures::*;
 pub mod error;
 pub use error::*;
 
-use crate::rng::FiatShamirRng;
+use crate::fiat_shamir_rng::FiatShamirRng;
 
 /// A random number generator that bypasses some limitations of the Rust borrow
 /// checker.
 pub mod optional_rng;
 
-/// A polynomial commitment scheme based on the hardness of the
-/// discrete logarithm problem in prime-order groups.
-/// The construction is detailed in [[BCMS20]][pcdas].
-///
-/// [pcdas]: https://eprint.iacr.org/2020/499
+/// The BCMS20 variant of the dlog commitment scheme.
 pub mod ipa_pc;
 
-/// `QuerySet` is the set of queries that are to be made to a set of labeled polynomials/equations
-/// `p` that have previously been committed to. Each element of a `QuerySet` is a pair of
-/// `(label, (point_label, point))`, where `label` is the label of a polynomial in `p`,
-/// `point_label` is the label for the point (e.g., "beta"), and  and `point` is the field element
-/// that `p[label]` is to be queried at.
+/// `QuerySet` is the set of queries that are to be made to a set of labeled polynomials or linear combinations
+///  Each element of a `QuerySet` is a pair of `(label, (point_label, point))`, where
+///     - `label` is the label of a polynomial in `p`,
+///     - `point_label` is the label for the point (e.g., "beta"), and  and
+///     - `point` is the field element that `p[label]` is to be queried at.
 pub type QuerySet<'a, F> = BTreeSet<(String, (String, F))>;
 
-/// `Evaluations` is the result of querying a set of labeled polynomials or equations
+/// `Evaluations` is the result of querying a set of labeled polynomials or linear combinations
 /// `p` at a `QuerySet` `Q`. It maps each element of `Q` to the resulting evaluation.
 /// That is, if `(label, query)` is an element of `Q`, then `evaluation.get((label, query))`
 /// should equal `p[label].evaluate(query)`.
 pub type Evaluations<'a, F> = BTreeMap<(String, F), F>;
 
 /// A proof of satisfaction of linear combinations.
 #[derive(Clone, CanonicalSerialize, CanonicalDeserialize)]
-pub struct BatchLCProof<F: Field, PC: PolynomialCommitment<F>> {
+pub struct BatchLCProof<G: AffineCurve, PC: PolynomialCommitment<G>> {
```

Comment:
We should rename it if possible. It is not a batch of proofs.

Reply:
Done, renamed to MultiPointProof